src/docfx/lib/schema/JsonSchemaValidation.cs

```diff
@@ -58,6 +58,16 @@ private static void ValidateScalar(JsonSchema schema, List<Error> errors, JValue
             {
                 errors.Add(Errors.UndefinedValue(JsonUtility.GetSourceInfo(scalar), scalar, schema.Enum));
             }
+
+            if ((schema.MaxLength.HasValue || schema.MinLength.HasValue) && scalar.Value is string str)
```

Comment:
do we need throw exception here if value is not string?

Reply:
tried that at "https://www.jsonschemavalidator.net/", no validation error thrown.